src/main.css

```diff
@@ -0,0 +1,174 @@
+/* Open Sans - Google font importing */
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
+
+/* Reset styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Global Styles */
+body {
+  background-color: #f6f6f6;
+  font-family: 'Open Sans', sans-serif;
+  text-align: center;
+  font-weight: 300;
+  font-size: 14px;
+  color: #1d1d1d;
+}
+
+ul li {
+  list-style: none;
+}
+
+main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* Todo Wrapper styles */
+.todo-wrapper {
+  width: 500px;
+  max-height: 660px;
+  overflow-y: scroll;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #fff;
+  border-radius: 3px;
+  box-shadow: 0 2px 8px rgb(0 0 0 / 15%);
+}
+
+.todo-wrapper::-webkit-scrollbar {
+  width: 0;
+}
+
+.todo-wrapper::-webkit-scrollbar-thumb {
+  background-color: #d3d3d3;
+}
+
+.todo-header {
+  display: flex;
+  justify-content: space-between;
+  padding: 16px;
+  border-bottom: 1px solid #d8d8d8;
+  position: sticky;
+  top: 0;
+  z-index: 999;
+  background-color: #fff;
+}
+
+.todo-header h2 {
+  font-size: 16px;
+}
+
+.todo-header svg {
+  width: 20px;
+  color: #b3b3b3;
+  cursor: pointer;
+}
+
+/* Add todo form styles */
+.add-todo-form {
+  width: 100%;
+  padding: 15px;
+  border-bottom: 1px solid #d8d8d8;
+}
+
+form {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+form input {
+  width: 95%;
+  border: none;
+  font-style: italic;
+  font-weight: lighter;
+  padding-left: 5px;
+}
+
+form input:focus {
+  outline: none;
+}
+
+form button[type='submit'] {
+  border: none;
+  background: none;
+}
+
+.material-icons,
+.enter-arrow {
+  color: #b3b3b3;
+  cursor: pointer;
+}
+
+/* Todo Lists Items */
+.todo-list {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.toggle {
+  display: flex;
+  justify-content: center;
+}
+
+.todo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #d8d8d8;
+  padding: 15px 0;
+  padding: 15px;
+}
+
+.check-div {
+  /* stylelint-disable-next-line csstree/validator */
```

Comment:
- [ ] Kindly remove this stylelint disable line its best to fix the issue in the code.

Reply:
Sure 👍It's fixed in the CSS file now. Thanks